src/content/docs/en/proyects-boilerplate.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Accelerate Your Development with the JDK Boilerplate"
+description: "Discover the power of the JDK Boilerplate in streamlining your development process. This comprehensive blog post guides you through cloning the repository, installing dependencies, and running the project. Learn about essential commands, notes, customizations, and components, as well as utilizing Astro and VS Code extensions for enhanced efficiency."
+image:
+  {
+    src: /images/carl-heyerdahl-KE0nC8-58MQ-unsplash.jpg,
+    attribution:Photo by <a href="https://unsplash.com/@carlheyerdahl?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Carl Heyerdahl</a> on <a href="https://unsplash.com/photos/KE0nC8-58MQ?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a>,
+  }
+---
+
+1. **Easy Setup and Installation**:
+   Clone the JDK Boilerplate repository and quickly set up your project with the provided commands. Learn how to install dependencies and run the project effortlessly.
+
+2. **Convenient Commands and Configurations**:
+   Explore the wide range of commands available to streamline your development workflow. From installing dependencies and building your production site to previewing your changes locally, leverage the power of predefined commands for increased productivity.
+
+3. **Customization and Component Creation**:
+   Learn how to customize the boilerplate to meet your project requirements. Create components with ease by following step-by-step instructions, and understand how to import styles, configure modal instances, and listen to modal events effectively.
+
+4. **Boosting Efficiency with Astro Extensions**:
+   Discover the Astro extension for Visual Studio Code, providing syntax highlighting and convenient code snippets. Install the Astro Snippets extension for quick and easy component creation, saving valuable time during development.
+
+5. **Ensuring Code Quality and Consistency**:
+   Utilize the provided ESLint, Prettier, and Stylelint configurations to maintain code quality and adhere to best practices. Learn about available resources for further reference on configuration options and style rules.
```

Comment:
At the end of this paragraph we should have a link to the resources that we mentioned here. Just to make sure they don't leave the site to google what are those things (in case they don't know anything at all about ESlint, Prettier, etc)